edit-profile.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Bootstrap 4 User Edit Form</title>
+    <meta name="description" content="Example user profile form and information editing form for Bootstrap 4." />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="generator" content="Codeply">
+
+
+
+    <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.4/css/bootstrap.min.css" />
+
+    <link rel="stylesheet" href="css/styles.css" />
+  </head>
+  <body >
+    <div class="container">
+    <div class="row m-y-2">
+        <!-- edit form column -->
+        <div class="col-lg-4 text-lg-center">
+            <h2>Edit Profile</h2>
+        </div>
+        <div class="col-lg-8">
+            <div class="alert alert-info alert-dismissable"> <a class="panel-close close" data-dismiss="alert">×</a>
+                Welcome! Please edit your profile below:
+            </div>
+        </div>
+        <div class="col-lg-8 push-lg-4 personal-info">
+             <form role="form">
+                <div class="form-group row">
+                    <label class="col-lg-3 col-form-label form-control-label">First name</label>
+                    <div class="col-lg-9">
+                        <input class="form-control" type="text" />
+                    </div>
+                </div>
+                <div class="form-group row">
+                    <label class="col-lg-3 col-form-label form-control-label">Last name</label>
+                    <div class="col-lg-9">
+                        <input class="form-control" type="text" />
+                    </div>
+                </div>
+                <div class="form-group row">
+                    <label class="col-lg-3 col-form-label form-control-label">Email</label>
+                    <div class="col-lg-9">
+                        <input class="form-control" type="email" />
+                    </div>
+                </div>
+                <div class="form-group row">
+                    <label class="col-lg-3 col-form-label form-control-label">Time Zone</label>

```

Comment:
I think the inclusion of time zones for the U.S. makes sense. This will probably mostly be a U.S. audience and knowing what timezone commenters are in will allow us to display localized times on comments.